CONTRIBUTING.md

```diff
@@ -16,17 +16,30 @@ so please explain why the changes need to be made (unless it is self-evident).
 * It is the responsibility of the author of the merge to ensure their merge is in a mergeable state.
 * It is the responsibility of the maintainers to ensure the merge process is unambiguous and automated where possible.
 
-### Branch naming
-Branch names should be of the format:
+### Branches
 
-`apm-nnn-short-issue-description`
+All changes are created on a short lived branch specifically for that change. Once ready, the branch must be merged into the default branch via a Pull Request.
```

Comment:
Do we want to say `release` instead of default branch?

Reply:
I pondered it and I would have used the actual name of the default branch if it wasn't called `release`. It can be confusing in my opinion.
I can change it if you want?

Reply:
Yeah I'd rather us be explicit with it, even if the branch naming is confusing 😆